src/main/kotlin/com/uber/nanoscope/Main.kt

```diff
@@ -219,7 +219,10 @@ class OpenHandler(private val args: List<String>): Runnable {
             exitProcess(1)
         }
 
-        Nanoscope.openTrace(inFile)
+        val sampleFile = File(args[0] + ".timer")
```

Comment:
This API is a bit weird - Can we take the sample files as options instead? ie: `--timer-data=...`, etc.

Reply:
Agree it's a bit weird. It's actually the same level of weirdness as the `adb` command to start nanoscope tracing: we only provides the path to the trace file and all other log names are generated automatically by appending extensions `.timer` and `.state`.

Another reason to choose this design is that I don't want to break any existing scripts using Nanoscope, i.e. people can just use the same commands they used before to start/stop profiling and visualization.

Reply:
Makes sense. Maybe we can make those optional flags and if they are not provided then nanoscope will try to add extensions and search them automatically? In this case if users(like me) are too lazy to provide additional flags they can jus use the old command.